### PR TITLE
[MIG] base_sparse_field: Nothing to do

### PR DESCRIPTION
**Pull request by hhgabelgaard**

### CODE REVIEW

**Review by legalsylvain — APPROVED** [review]

File: openupgrade_scripts/scripts/base_sparse_field/15.0.1.0/upgrade_analysis.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@
 ---Fields in module 'base_sparse_field'---
 ---XML records in module 'base_sparse_field'---
 ---nothing has changed in this module--
+# Nothing to do
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/base_sparse_field/15.0.1.0/upgrade_analysis.txt:5]: This should be done in a copied file `upgrade_analysis_work.txt`

**Comment by pedrobaeza** [comment]
@legalsylvain please check details before merging.

File: openupgrade_scripts/scripts/base_sparse_field/15.0.1.0/upgrade_analysis.txt (hunk 1 above)
Comment (hhgabelgaard) [reply openupgrade_scripts/scripts/base_sparse_field/15.0.1.0/upgrade_analysis.txt:5]: Thanks.
Will do

**Pull request merged**